src/controllers/auth/login.js

```diff
@@ -56,29 +56,6 @@ define(["apphost", "appSettings", "dom", "connectionManager", "loading", "layout
         }
     }
 
-    var metroColors = ["#6FBD45", "#4BB3DD", "#4164A5", "#E12026", "#800080", "#E1B222", "#008040", "#0094FF", "#FF00C7", "#FF870F", "#7F0037"];
-
-    function getRandomMetroColor() {
-        var index = Math.floor(Math.random() * (metroColors.length - 1));
-        return metroColors[index];
-    }
-
-    function getMetroColor(str) {
```

Comment:
well, that function at least gave users a different (but stable) colors... maybe we should do some colored outlines, too, instead of completely dropping this and forcing users to either upload avatars or all look exactly the same?

Reply:
We could use the color generator in cardBuilder to pick a color in the theme from one of the 5 defaultBackground classes, based on the name of the user.